lib/ramble/ramble/cmd/workspace.py

```diff
@@ -243,8 +243,8 @@ def _workspace_create(name_or_path, dir=False,
 
     if template_execute:
         with open(template_execute, 'r') as f:
-            workspace._read_execution_template(f.read())
-            workspace._write_execution_template()
+            workspace._read_template('execute_experiment', f.read())
```

Comment:
Does it make sense to get the name of the template from the file, instead of creating it as `execute_experiment`?

It would be kind of sad if someone used:
`ramble workspace create -t /tmp/my_awesome_template.tpl -c ramble.yaml`

and referenced `{my_aweseom_template}` in `ramble.yaml`, but this broke that. I'm not sure I feel like this is an important thing to do right now. So, if it's difficult, it might not be worth the effort.

Reply:
Yea that's a good idea, I'll add it